src/core/tracking.js

```diff
@@ -105,8 +105,23 @@ export function getAction(type) {
   }[type] || TRACKING_TYPE_INVALID;
 }
 
+export function doNotTrackEnabled({
+  _navigator = typeof navigator !== 'undefined' ? navigator : null,
+  _window = typeof window !== 'undefined' ? window : null,
+} = {}) {
+  if (!_navigator || !_window) {
+    return false;
+  }
+
+  const dnt = _navigator.doNotTrack || _window.doNotTrack;
+  return dnt === '1';
```

Comment:
All the docs indicate as much, and my testing in Safari, Chrome, and Firefox say yes https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack#Browser_compatibility

Gonna test Edge later as my Windows PC is doing an update dance. 😓